Resources/Prototypes/_ES/Objectives/crew.yml

```diff
@@ -111,3 +112,46 @@
     title: es-guzzle-unique-reagents-objective-title
     target: 15
     maxTarget: 25
+
+- type: entity
+  parent: ESBaseMaskObjective
+  id: ESObjectiveTakeDamage
+  name: TakeDamage
+  description: Take as much of a certain damage type as you can.
+  components:
+  - type: ESObjective
+    icon:
+      sprite: Objects/Weapons/Melee/baseball_bat.rsi
+      state: icon
+  - type: ESTakeDamageObjective
+    descriptionLoc: es-daredevil-desc
+    requiredDamages:
+    - Brute
+    - Burn
+    - Airloss
+    - Airloss
```

Comment:
for the damage types, I kinda want them to be more noteworthy. Brute, burn, and airloss are all fairly common and i think that detracts from being interesting. I'd like to see electric, cold, etc. More specific types rather than just general groups